spec/integrations/base_spec.rb

```diff
@@ -58,6 +58,20 @@
       end
     end
 
+    context 'Given passed object is an instance of a configured array-like class' do
+      let(:blueprint) { blueprint_with_block }
+      let(:additional_object) { OpenStruct.new(obj_hash.merge(id: 2)) }
+      let(:obj) { Set.new([object_with_attributes, additional_object]) }
+
+      before do
+        Blueprinter.configure { |config| config.array_like_classes = [Set] }
+      end
+
+      it 'should return the expected array of hashes' do
+        should eq('[{"id":1,"position_and_company":"Manager at Procore"},{"id":2,"position_and_company":"Manager at Procore"}]')
+      end
+    end
```

Comment:
can we also add a negative case for this? what happens when I try to pass an Array like object but it's not added to `array_like_classes`?

Reply:
sure thing, I'll ping you after I've added

Reply:
the only thing to assert here though is that it raises a NoMethodError, which is what it currently does before this PR and still does after this PR.

Reply:
I added a test case for it raising an error.